composer.json

```diff
@@ -23,10 +23,16 @@
             "name": "Daniel Wittberger",
             "email": "daniel@wittberger.net",
             "role": "Maintainer"
+        },
+        {
+            "name": "Judith Jeszenkovits",
+            "email": "judith.jeszenkovits@gmail.com",
+            "role": "Maintainer"
         }
     ],
     "require": {
         "php": "^7.4",
+        "illuminate/support": "^7.5",
```

Comment:
Is it necessary that we use the entire illuminate/support library? 

I think this one will be much lighter: https://github.com/tightenco/collect

Reply:
maybe we can get rid of the dependency at all and implement it with an array?

Reply:
We talked about getting rid of the collections. But we decided to stick with the collections but replace illuminate/support with the more lightweight tightenco/collect.